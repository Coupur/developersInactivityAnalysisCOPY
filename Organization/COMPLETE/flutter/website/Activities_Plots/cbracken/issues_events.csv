id,date,event,creator_login
2239861306,2019-03-29 16:57:33,review_requested,cbracken
2239861313,2019-03-29 16:57:33,review_requested,cbracken
2239881021,2019-03-29 17:04:30,head_ref_force_pushed,cbracken
2239883460,2019-03-29 17:05:14,head_ref_force_pushed,cbracken
2239891929,2019-03-29 17:08:01,head_ref_force_pushed,cbracken
2239929825,2019-03-29 17:22:02,referenced,cbracken
2239929826,2019-03-29 17:22:02,merged,cbracken
2239929828,2019-03-29 17:22:02,closed,cbracken
2239930057,2019-03-29 17:22:07,head_ref_deleted,cbracken
2033057890,2018-12-18 14:03:58,mentioned,cbracken
2033057893,2018-12-18 14:03:58,subscribed,cbracken
2041431126,2018-12-21 18:46:56,mentioned,cbracken
2041431128,2018-12-21 18:46:56,subscribed,cbracken
2033015711,2018-12-18 13:47:02,mentioned,cbracken
2033015712,2018-12-18 13:47:02,subscribed,cbracken
2030513885,2018-12-17 15:12:24,mentioned,cbracken
2030513886,2018-12-17 15:12:24,subscribed,cbracken
2030388959,2018-12-17 14:26:18,mentioned,cbracken
2030388961,2018-12-17 14:26:18,subscribed,cbracken
2058887492,2019-01-07 23:21:05,closed,cbracken
2014442538,2018-12-09 15:46:00,mentioned,cbracken
2014442539,2018-12-09 15:46:00,subscribed,cbracken
2017176514,2018-12-10 22:04:01,mentioned,cbracken
2017176515,2018-12-10 22:04:01,subscribed,cbracken
2029082282,2018-12-16 23:01:08,mentioned,cbracken
2029082283,2018-12-16 23:01:08,subscribed,cbracken
2003493683,2018-12-04 07:38:04,mentioned,cbracken
2003493684,2018-12-04 07:38:04,subscribed,cbracken
1933968130,2018-10-30 03:22:50,mentioned,cbracken
1933968131,2018-10-30 03:22:50,subscribed,cbracken
1833836500,2018-09-07 22:03:39,review_requested,cbracken
1833836504,2018-09-07 22:03:39,review_requested,cbracken
1833836505,2018-09-07 22:03:39,review_requested,cbracken
1833895879,2018-09-07 22:53:32,head_ref_force_pushed,cbracken
1836928206,2018-09-10 20:02:47,referenced,cbracken
1836928209,2018-09-10 20:02:47,merged,cbracken
1836928217,2018-09-10 20:02:47,closed,cbracken
1836949860,2018-09-10 20:13:02,head_ref_deleted,cbracken
1833830267,2018-09-07 21:59:41,review_requested,cbracken
1833830268,2018-09-07 21:59:41,review_requested,cbracken
1833830269,2018-09-07 21:59:41,review_requested,cbracken
1833837536,2018-09-07 22:04:24,head_ref_force_pushed,cbracken
1833897162,2018-09-07 22:54:44,head_ref_force_pushed,cbracken
1836948204,2018-09-10 20:12:09,base_ref_changed,cbracken
1836953425,2018-09-10 20:14:44,referenced,cbracken
1836953426,2018-09-10 20:14:44,merged,cbracken
1836953427,2018-09-10 20:14:44,closed,cbracken
1836953494,2018-09-10 20:14:47,head_ref_deleted,cbracken
1837120919,2018-09-10 21:38:30,mentioned,cbracken
1837120920,2018-09-10 21:38:30,subscribed,cbracken
1751627804,2018-07-25 03:26:30,mentioned,cbracken
1751627805,2018-07-25 03:26:30,subscribed,cbracken
1640063890,2018-05-22 18:33:43,mentioned,cbracken
1640063891,2018-05-22 18:33:43,subscribed,cbracken
1630389613,2018-05-16 19:13:48,mentioned,cbracken
1630389615,2018-05-16 19:13:48,subscribed,cbracken
1630398248,2018-05-16 19:18:29,mentioned,cbracken
1630398249,2018-05-16 19:18:29,subscribed,cbracken
1654181979,2018-05-30 17:44:31,mentioned,cbracken
1654181980,2018-05-30 17:44:31,subscribed,cbracken
1564651841,2018-04-09 19:50:45,mentioned,cbracken
1564651844,2018-04-09 19:50:45,subscribed,cbracken
1552172754,2018-04-02 15:37:37,mentioned,cbracken
1552172756,2018-04-02 15:37:37,subscribed,cbracken
1498335705,2018-03-01 06:44:15,referenced,cbracken
1498335706,2018-03-01 06:44:15,merged,cbracken
1498335707,2018-03-01 06:44:15,closed,cbracken
1498335798,2018-03-01 06:44:19,head_ref_deleted,cbracken
1044159851,2017-04-15 18:02:09,review_requested,cbracken
1044174924,2017-04-15 19:00:48,referenced,cbracken
1044174925,2017-04-15 19:00:48,merged,cbracken
1044174926,2017-04-15 19:00:48,closed,cbracken
1044174931,2017-04-15 19:00:50,head_ref_deleted,cbracken
1044156657,2017-04-15 17:47:12,review_requested,cbracken
1044156658,2017-04-15 17:47:12,review_requested,cbracken
1044156819,2017-04-15 17:48:06,head_ref_force_pushed,cbracken
1044159203,2017-04-15 17:59:43,referenced,cbracken
1044159204,2017-04-15 17:59:43,merged,cbracken
1044159205,2017-04-15 17:59:43,closed,cbracken
1044159212,2017-04-15 17:59:46,head_ref_deleted,cbracken
1044146720,2017-04-15 17:07:27,review_requested,cbracken
1044146721,2017-04-15 17:07:27,review_requested,cbracken
1044150821,2017-04-15 17:24:03,head_ref_force_pushed,cbracken
1044156568,2017-04-15 17:46:50,referenced,cbracken
1044156569,2017-04-15 17:46:50,merged,cbracken
1044156570,2017-04-15 17:46:50,closed,cbracken
1044156584,2017-04-15 17:46:52,head_ref_deleted,cbracken
1044142568,2017-04-15 16:51:01,review_requested,cbracken
1044142569,2017-04-15 16:51:01,review_requested,cbracken
1044142570,2017-04-15 16:51:01,review_requested,cbracken
1044150587,2017-04-15 17:23:05,referenced,cbracken
1044150588,2017-04-15 17:23:05,merged,cbracken
1044150589,2017-04-15 17:23:05,closed,cbracken
1044150608,2017-04-15 17:23:09,head_ref_deleted,cbracken
1044134033,2017-04-15 16:13:01,review_requested,cbracken
1044134035,2017-04-15 16:13:01,review_requested,cbracken
1044175622,2017-04-15 19:02:30,head_ref_force_pushed,cbracken
1044177237,2017-04-15 19:08:31,referenced,cbracken
1044177238,2017-04-15 19:08:31,merged,cbracken
1044177240,2017-04-15 19:08:31,closed,cbracken
1044177253,2017-04-15 19:08:34,head_ref_deleted,cbracken
1019055391,2017-03-28 18:03:59,mentioned,cbracken
1019055392,2017-03-28 18:03:59,subscribed,cbracken
1018996549,2017-03-28 17:29:30,review_requested,cbracken
1018996760,2017-03-28 17:29:37,review_requested,cbracken
1019104376,2017-03-28 18:31:30,head_ref_force_pushed,cbracken
1019197538,2017-03-28 19:28:51,referenced,cbracken
1019197539,2017-03-28 19:28:51,merged,cbracken
1019197540,2017-03-28 19:28:51,closed,cbracken
1019197600,2017-03-28 19:28:54,head_ref_deleted,cbracken
1015178453,2017-03-25 00:50:53,closed,cbracken
1044176099,2017-04-15 19:04:00,head_ref_deleted,cbracken
1013486094,2017-03-24 00:54:12,review_requested,cbracken
1013486095,2017-03-24 00:54:12,review_requested,cbracken
1017083744,2017-03-27 17:14:24,head_ref_force_pushed,cbracken
1017714422,2017-03-28 00:52:33,head_ref_force_pushed,cbracken
1018852400,2017-03-28 16:04:54,mentioned,cbracken
1018852401,2017-03-28 16:04:54,subscribed,cbracken
1018951977,2017-03-28 17:01:55,referenced,cbracken
1018951978,2017-03-28 17:01:55,merged,cbracken
1018951979,2017-03-28 17:01:55,closed,cbracken
1018952098,2017-03-28 17:01:58,head_ref_deleted,cbracken
1003823847,2017-03-17 02:35:19,mentioned,cbracken
1003823848,2017-03-17 02:35:19,subscribed,cbracken
995716127,2017-03-10 23:21:28,head_ref_force_pushed,cbracken
997646595,2017-03-13 17:17:51,head_ref_force_pushed,cbracken
997664787,2017-03-13 17:27:42,referenced,cbracken
997664788,2017-03-13 17:27:42,merged,cbracken
997664789,2017-03-13 17:27:42,closed,cbracken
997664866,2017-03-13 17:27:44,head_ref_deleted,cbracken
992578701,2017-03-09 05:51:38,mentioned,cbracken
992578702,2017-03-09 05:51:38,subscribed,cbracken
992612226,2017-03-09 06:35:18,referenced,cbracken
992612227,2017-03-09 06:35:18,merged,cbracken
992612229,2017-03-09 06:35:18,closed,cbracken
992612254,2017-03-09 06:35:20,head_ref_deleted,cbracken
992566346,2017-03-09 05:33:59,head_ref_force_pushed,cbracken
992566415,2017-03-09 05:34:04,mentioned,cbracken
992566416,2017-03-09 05:34:04,subscribed,cbracken
992570275,2017-03-09 05:39:41,referenced,cbracken
992570276,2017-03-09 05:39:41,merged,cbracken
992570277,2017-03-09 05:39:41,closed,cbracken
992570290,2017-03-09 05:39:43,head_ref_deleted,cbracken
994253695,2017-03-10 02:40:30,mentioned,cbracken
994253697,2017-03-10 02:40:30,subscribed,cbracken
995731813,2017-03-10 23:38:45,mentioned,cbracken
995731814,2017-03-10 23:38:45,subscribed,cbracken
995760954,2017-03-11 00:17:59,mentioned,cbracken
995760955,2017-03-11 00:17:59,subscribed,cbracken
995768980,2017-03-11 00:30:40,mentioned,cbracken
995768981,2017-03-11 00:30:40,subscribed,cbracken
986318378,2017-03-04 02:33:08,mentioned,cbracken
986318379,2017-03-04 02:33:08,subscribed,cbracken
976539986,2017-02-24 19:15:04,mentioned,cbracken
976539988,2017-02-24 19:15:04,subscribed,cbracken
963622225,2017-02-15 18:54:58,referenced,cbracken
963622226,2017-02-15 18:54:58,merged,cbracken
963622227,2017-02-15 18:54:58,closed,cbracken
963622547,2017-02-15 18:55:12,head_ref_deleted,cbracken
947370665,2017-02-03 00:12:15,referenced,cbracken
947370667,2017-02-03 00:12:15,merged,cbracken
947370670,2017-02-03 00:12:15,closed,cbracken
947370711,2017-02-03 00:12:18,head_ref_deleted,cbracken
890722392,2016-12-12 17:31:38,head_ref_deleted,cbracken
846841604,2016-11-03 18:15:19,referenced,cbracken
846841605,2016-11-03 18:15:19,merged,cbracken
846841606,2016-11-03 18:15:19,closed,cbracken
1044175952,2017-04-15 19:03:20,head_ref_deleted,cbracken
