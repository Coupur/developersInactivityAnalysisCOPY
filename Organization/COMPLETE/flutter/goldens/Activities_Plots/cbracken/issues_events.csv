id,date,event,creator_login
1728648718,2018-07-11 18:36:44,referenced,cbracken
1728648719,2018-07-11 18:36:44,merged,cbracken
1728648720,2018-07-11 18:36:44,closed,cbracken
1728648880,2018-07-11 18:36:48,head_ref_deleted,cbracken
1726441354,2018-07-10 20:11:57,referenced,cbracken
1726441355,2018-07-10 20:11:57,merged,cbracken
1726441357,2018-07-10 20:11:57,closed,cbracken
1726441473,2018-07-10 20:12:02,head_ref_deleted,cbracken
1726238821,2018-07-10 18:25:33,closed,cbracken
1726238823,2018-07-10 18:25:33,head_ref_force_pushed,cbracken
1726442193,2018-07-10 20:12:23,head_ref_deleted,cbracken
1726250802,2018-07-10 18:31:19,referenced,cbracken
1726446550,2018-07-10 20:14:43,referenced,cbracken
1726441867,2018-07-10 20:12:14,head_ref_deleted,cbracken
1726441924,2018-07-10 20:12:16,head_ref_deleted,cbracken
